stripe/res/layout/card_input_widget.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Start with a `merge` because this is going to be inflated in a class that extends `LinearLayout`, so we lose one layer already